helm-chart/configs/mesh-config.conf

```diff
@@ -9,13 +9,8 @@
     "clientMaxBodySize": {{ quote .Values.clientMaxBodySize }},
     "environment": {{ quote .Values.environment }},
     "isUDPEnabled": {{ .Values.enableUDP }},
-    "enabledNamespaces": [{{ if .Values.enabledNamespaces }} {{ range $idx, $elem := .Values.enabledNamespaces }}{{if $idx}},{{end}}{{quote .}}{{end}} {{ else }} {{ range $idx, $elem := .Values.autoInjection.enabledNamespaces }}{{if $idx}},{{end}}{{quote .}}{{end}} {{ end }}],
-    "isAutoInjectEnabled": {{ if eq .Values.disableAutoInjection .Values.autoInjection.disable }} {{ not .Values.disableAutoInjection }} {{ else if .Values.disableAutoInjection }} {{ not .Values.disableAutoInjection }} {{ else }} {{ not .Values.autoInjection.disable }} {{ end }},
-    "injection": {
-      "disabledNamespaces": [{{ range $idx, $elem := .Values.autoInjection.disabledNamespaces }}{{if $idx}},{{end}}{{quote .}}{{end}}],
-      "enabledNamespaces": [{{ if .Values.enabledNamespaces }} {{ range $idx, $elem := .Values.enabledNamespaces }}{{if $idx}},{{end}}{{quote .}}{{end}} {{ else }} {{ range $idx, $elem := .Values.autoInjection.enabledNamespaces }}{{if $idx}},{{end}}{{quote .}}{{end}} {{ end }}],
-      "isAutoInjectEnabled": {{ if eq .Values.disableAutoInjection .Values.autoInjection.disable }} {{ not .Values.disableAutoInjection }} {{ else if .Values.disableAutoInjection }} {{ not .Values.disableAutoInjection }} {{ else }} {{ not .Values.autoInjection.disable }} {{ end }}
-    },
+    "enabledNamespaces": [{{ if .Values.enabledNamespaces }} {{ range $idx, $elem := .Values.enabledNamespaces }}{{if $idx}},{{end}}{{quote .}}{{end}} {{ end }}],
```

Comment:
I'm curious if the `if` statement is needed? What's the behavior if the list is empty or not provided, and we don't have that statement in there?

Reply:
Took this out for now, seems like nothing happens without the if statement when the list is not provided